rh/schema.py

```diff
@@ -26,6 +37,34 @@ def get_slack_notify_text(rhr: models.RentalHistoryRequest) -> str:
     return f"{user_text} has requested {rh_link}!"
 
 
+def run_rent_stab_sql_query(bbl: str) -> Optional[Dict[str, Any]]:
+    sql_query = """
+        select uc2007, uc2008, uc2009, uc2010, uc2011, uc2012, uc2013,
+             uc2014, uc2015, uc2016, uc2017, uc2018, uc2019
+        from rentstab
+        full join rentstab_v2 using(ucbbl)
+        where ucbbl = %(bbl)s
+    """
+    if not settings.NYCDB_DATABASE:
+        return None
+
+    with connections[settings.NYCDB_DATABASE].cursor() as cursor:
+        cursor.execute(sql_query, {"bbl": bbl})
+        json_result = list(generate_json_rows(cursor))
+        if not json_result:
+            return BLANK_RENT_STAB_INFO
+        return json_result[0]
+
+
+def process_rent_stab_data(raw_data: Optional[Dict[str, Any]]) -> Optional[Dict[str, Any]]:
+    if not raw_data:
+        return BLANK_RENT_STAB_INFO
```

Comment:
Er, shouldn't this return `None` if `raw_data` is `None`?  It seems like `raw_data` is only going to be `None` if NYCDB isn't configured... hmm.

Actually it feels confusing to have a function that takes an optional argument for the sole purpose of returning `None` when it's passed `None`.  Maybe this function should take a non-optional `Dict[str, Any]` and always return a non-optional `Dict[str, Any]` instead, and the caller should be in charge of only calling the function if it has actual `raw_data` to provide?

It seems like perhaps a _third_ helper function would be useful here, one called `get_rent_stab_data_for_bbl` which takes a BBL and returns an `Optional[Dict[str, Any]]`...